src/block/countdown/divider.js

```diff
@@ -0,0 +1,240 @@
+	const hasDivider = useBlockAttributesContext( attributes => attributes.hasDivider )
+	const dividerType = useBlockAttributesContext( attributes => attributes.dividerType )
+	const setAttributes = useBlockSetAttributesContext()
+
+	return (
+		<InspectorStyleControls>
+			<PanelAdvancedSettings
+				title={ __( 'Separator', i18n ) }
+				id="divider"
+				hasToggle={ true }
+				checked={ hasDivider }
+				onChange={ hasDivider => setAttributes( { hasDivider } ) }
+			>
+				<AdvancedToolbarControl
+					controls={ DIVIDER_TYPE_CONTROLS }
+					attribute="dividerType"
+					fullwidth={ true }
+					default={ ':' }
+					isSmall={ false }
+				/>
+				{ dividerType === ':' && <AdvancedRangeControl
+					label={ __( 'Size', i18n ) }
+					min={ 1 }
+					sliderMin={ 1 }
+					sliderMax={ 100 }
+					attribute="dividerSizeColon"
+					placeholder="45"
+				/> }
+				{ dividerType === ':' && <AdvancedRangeControl
+					label={ __( 'Top Offset', i18n ) }
+					min={ -50 }
+					sliderMin={ -50 }
+					sliderMax={ 50 }
+					attribute="dividerTopOffset"
+					placeholder="-12"
+				/> }
+				{ dividerType === '|' && <AdvancedRangeControl
+					label={ __( 'Size', i18n ) }
+					min={ 1 }
+					max={ 100 }
+					sliderMin={ 1 }
+					sliderMax={ 100 }
+					attribute="dividerSizeLine"
+					placeholder="50"
+				/> }
+				<ColorPaletteControl
+					label={ __( 'Color', i18n ) }
+					attribute="dividerColor"
+				/>
+			</PanelAdvancedSettings>
+		</InspectorStyleControls>
+	)
+}
+
+export const Divider = props => {
+	const {
+		attributes,
+	} = props
+
+	return (
+		<div className="stk-block-countdown__divider-wrapper">
+			{ attributes.dividerType === ':' &&
+				( <div className="stk-block-countdown__divider-colon">
+					<> : </>
+				</div>
+				)
+			}
+			{ attributes.dividerType === '|' &&
+				( <div className="stk-block-countdown__divider-line">
+				</div>
+				)
+			}
+		</div>
+
+	)
+}
+
+Divider.Content = props => {
+	const {
+		attributes,
+		className,
+	} = props
+
+	return (
+		<div className="stk-block-countdown__divider-wrapper">
+			<div className={ className }>
+				{ attributes.dividerType === ':' &&
+				( <div className={ className }>
+					<> : </>
+				</div>
+				)
+				}
+				{ attributes.dividerType === '|' &&
+				( <div className={ className }>
+					<> : </>
+				</div>
+				)
+				}
+			</div>
+		</div>
+	)
+}
+
+const Styles = props => {
+	const propsToPass = {
+		...props,
+		version: props.version,
+		versionAdded: '3.0.0',
+		versionDeprecated: '',
+	}
+
+	return (
+		<>
+			{ <BlockCss
+				{ ...propsToPass }
+				selectorCallback={ getAttribute => getAttribute( 'dividerType' ) === ':' ? '.stk-block-countdown__divider-colon' : '.stk-block-countdown__divider-line' }
+				styleRuleCallback={ getAttribute => getAttribute( 'dividerType' ) === ':' ? 'color' : 'backgroundColor' }
+				attrName="dividerColor"
+				key="dividerColor"
+				responsive="all"
+				dependencies={ [ 'dividerType', 'dividerColor' ] }
```

Comment:
No need to include the value used in `attrName` as a dependency:

```suggestion
				dependencies={ [ 'dividerType' ] }
```

Please do this for the others.